katalog/kong/upstream/kustomization.yaml

```diff
@@ -1,7 +1,12 @@
+# Copyright (c) 2021 SIGHUP s.r.l All rights reserved.
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file.
+
+---
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 
 bases:
-  - github.com/Kong/kubernetes-ingress-controller/deploy/manifests/base?ref=0.8.1
+  - github.com/Kong/kubernetes-ingress-controller/deploy/manifests/base?ref=1.3.1
```

Comment:
Mixed feelings on using directly this base, i would put all the manifests in this repository, and maybe we can write somewhere that these are taken from github official repo version x.x.x